docker-swarm.yml

```diff
@@ -0,0 +1,41 @@
+version: '3.4'
```

Comment:
Can you give a tldr on this file or how to use it (maybe add to readme)? Not familiar with docker-swarm.